### PR TITLE
Add security-spring-boot-starter

### PR DESCRIPTION
**Pull request by persapiens**

### CODE REVIEW

File: joinfaces-starters/security-spring-boot-starter/build.gradle [hunk 1]
```diff
@@ -0,0 +1,13 @@
+description = 'Security Spring Boot Starter'
+jar.manifest.attributes('Automatic-Module-Name': 'joinfaces.starter.security')
+
+dependencies {
+    compile project(':joinfaces-starters:jsf-spring-boot-starter')
+    compile project(':spring-security-jsf-taglib')
```
Comment (larsgrefer) [inline joinfaces-starters/security-spring-boot-starter/build.gradle]: Should we rename the taglib to `joinfaces-security-taglib` to better match the other artifact names?

Comment (persapiens) [reply joinfaces-starters/security-spring-boot-starter/build.gradle]: yes.

Comment (persapiens) [reply joinfaces-starters/security-spring-boot-starter/build.gradle]: I've renamed spring-security-jsf-taglib to joifnaces-security-taglib.

File: src/docs/asciidoc/_features.adoc [hunk 2]
```diff
@@ -38,6 +38,9 @@ You can use the table below to discover the starters that fit your needs.
 |jsf-spring-boot-starter
 |Meta starter that pulls `tomcat-spring-boot-starter` and `mojarra-spring-boot-starter`.
 
+|security-spring-boot-starter
+|Security starter. Pulls `jsf-spring-boot-starter`, `spring-boot-starter-security` and `spring-security-jsb-taglib`.
```
Comment (larsgrefer) [inline src/docs/asciidoc/_features.adoc]:
```suggestion
|Security starter. Pulls `jsf-spring-boot-starter`, `spring-boot-starter-security` and `joinfaces-security-taglib`.
```

**Pull request merged**